src/plugins/ui_stateful/clipboard.ts

```diff
@@ -46,6 +46,7 @@ export class ClipboardPlugin extends UIPlugin {
   private lastPasteState?: ClipboardState;
   private _isPaintingFormat: boolean = false;
   private originSheetId?: UID;
+  private isPaintFormatPersistent: boolean = false;
```

Comment:
what do you think about merging both `_isPaintingFormat` and `isPaintFormatPersistent` into a single string property that would only take 3 values ?
Currently, the state `_isPaintingFormat = false` and `isPaintFormatPersistent = true` is technically possible even if it makes no sense and should never happen. With a single property and only 3 possible values, the inconsistent state is impossible by design (make impossible state impossible ;))

We need to come up with nice name though. The first thing coming to my mind is: `"inactive" | "active" | "persistent"` but I don't think those names are really good